nautobot_ssot/integrations/aristacv/utils/nautobot.py

```diff
@@ -59,6 +60,7 @@ def verify_device_role_object(role_name, role_color):
         role_obj = Role.objects.get(name=role_name)
     except Role.DoesNotExist:
         role_obj = Role(name=role_name, color=role_color)
+        role_obj.content_types.add(ContentType.objects.get_for_model(Device))
         role_obj.validated_save()
```

Comment:
```suggestion
        role_obj.validated_save()
        role_obj.content_types.add(ContentType.objects.get_for_model(Device))
```

I think we need to save before manipulating the many to many relationship